src/wasm/Cargo.toml

```diff
@@ -7,8 +7,13 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-# deno_config = { git = "https://github.com/denoland/deno_config.git", rev = "59e578eaea0b720b4d79d5617b1f2a1c076b0d2b" }
+deno_config = "0.36.0"
```

Comment:
I recommend bumping to 0.37.1 before merging because it fixes an issue with workspaces not working with a specified config.